plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -349,7 +349,12 @@ private class DokkaDescriptorVisitor( // TODO: close this class and make it priv
             type = descriptor.type.toBound(),
             documentation = descriptor.resolveDescriptorData(platformData),
             platformData = listOf(platformData),
-            extra = descriptor.additionalExtras()
+            extra = descriptor.getDefaultValue().let {
+                if (it == null)
+                    descriptor.additionalExtras()
+                else
+                    descriptor.additionalExtras() + DefaultValue(it)
+            }
```

Comment:
```suggestion
            extra = descriptor.getDefaultValue()?.let {
                    descriptor.additionalExtras() + DefaultValue(it)
            } ?: descriptor.additionalExtras()
```
Checking for nullability in `.let` cannot succeed as `.let` will just fail when `getDefaultValue()` returns null

Reply:
Changed.